docs/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix.md

```diff
@@ -43,6 +43,9 @@ The analyzer with code fix template creates five projects:
 - **MakeConst.Test**, which is a unit test project.
 - **MakeConst.Vsix**, which is the default startup project that starts a second instance of Visual Studio that has loaded your new analyzer. Press <kbd>F5</kbd> to start the VSIX project.
 
+> [!NOTE]
+> Analyzers should target .NET Standard 2.0 because they can run in .NET Core environment (command line builds) and .NET Framework enviroenment (Visual Studio).
```

Comment:
typo:

```suggestion
> Analyzers should target .NET Standard 2.0 because they can run in .NET Core environment (command line builds) and .NET Framework environment (Visual Studio).
```